core-bundle/src/Command/DebugPagesCommand.php

```diff
@@ -86,13 +86,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
             $rows[] = [
                 $type,
-                $config ? $config->getPath() : '*',
-                $config ? $config->getUrlSuffix() : '*',
+                $config ? $config->getPath() ?: '-' : '*',
+                $config ? $config->getUrlSuffix() ?: '-' : '*',
```

Comment:
I'm not sure whether an empty path & suffix should be indicated with `-` (clear that the config doesn't have these) or `*` (default value with no config).

Reply:
I'd vote for `-`.